### PR TITLE
Added links to hashtags

### PR DESCRIPTION
**Pull request by sangeetagupta2068**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
 
-<p align="center">Passionate for <b>#opensource #code #github #remote #startup </b> & <b>#inclusion</b>. In recent years I have trended on GitHub with the likes of Twitter, Facebook, Angular etc and was in the top 500 open source contributors in the world. As a senior software engineer, the more I learn, the more I realise how much I still need to learn.</p>
+<p align="center">Passionate for <b> <u> <a href = "https://twitter.com/hashtag/opensource/"> #opensource </a> </u> <u> <a href = "https://twitter.com/hashtag/code/"> #code </a> </u> <u> <a href = "https://twitter.com/hashtag/github/"> #github </u> <u> <a href = "https://twitter.com/hashtag/remote/"> #remote </u> <u> <a href = "https://twitter.com/hashtag/startup/"> #startup </u> </a> </b> & <b> <u> <a href = "https://twitter.com/hashtag/inclusion/"> #inclusion </a> </u> </b>. In recent years I have trended on GitHub with the likes of Twitter, Facebook, Angular etc and was in the top 500 open source contributors in the world. As a senior software engineer, the more I learn, the more I realise how much I still need to learn.</p>
```
Comment (eddiejaoude) [inline README.md:5]: Looks good 👍 . I think the extra spaces between `href = "` are not needed. Maybe also spread it over the multiple lines, for examples...

```
<p align="center">Passionate for 
   <b><u><a href="https://twitter.com/hashtag/opensource/">#opensource</a> 
   ...
```

**Comment by eddiejaoude** [comment]
Looks good, a couple of suggestions 🤓

**Comment by sangeetagupta2068** [comment]
Sure, I will do it and submit a new PR!

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+<p align="center">Passionate for <b> <u> <a href="https://twitter.com/hashtag/opensource/"> #opensource </a> </u> <u> <a href="https://twitter.com/hashtag/code/"> #code </a> </u> <u> <a href="https://twitter.com/hashtag/github/"> #github </u> <u> <a href="https://twitter.com/hashtag/remote/"> #remote </u> <u> <a href="https://twitter.com/hashtag/startup/"> #startup </u> </a> </b> & <b> <u> <a href="https://twitter.com/hashtag/inclusion/"> #inclusion </a> </u> </b>. In recent years I have trended on GitHub with the likes of Twitter, Facebook, Angular etc and was in the top 500 open source contributors in the world. As a senior software engineer, the more I learn, the more I realise how much I still need to learn.</p>
```
Comment (eddiejaoude) [inline README.md]: You are missing a html closing `</a>` tag and on another link the `</u>` is not nested correctly . It is easier to see when spread over separate lines...

```html
<p align="center">
   Passionate for
   <b>
      <u><a href="https://twitter.com/hashtag/opensource/">#opensource</a></u>
      <u><a href="https://twitter.com/hashtag/code/">#code</a></u>
      <u><a href="https://twitter.com/hashtag/github/">#github</u>
      <u><a href="https://twitter.com/hashtag/remote/">#remote</u>
      <u><a href="https://twitter.com/hashtag/startup/">#startup</u></a>
   </b> 
   &
   <b>
      <u><a href="https://twitter.com/hashtag/inclusion/">#inclusion</a></u>
   </b>. In recent years I have trended on GitHub with the likes of Twitter, Facebook, Angular etc and was in the top 500 open source contributors in the world. As a senior software engineer, the more I learn, the more I realise how much I still need to learn.</p>
```

```suggestion
<p align="center">
   Passionate for
   <b>
      <u><a href="https://twitter.com/hashtag/opensource/">#opensource</a></u>
      <u><a href="https://twitter.com/hashtag/code/">#code</a></u>
      <u><a href="https://twitter.com/hashtag/github/">#github</a></u>
      <u><a href="https://twitter.com/hashtag/remote/">#remote</a></u>
      <u><a href="https://twitter.com/hashtag/startup/">#startup</a></u>
   </b> 
   &
   <b>
      <u><a href="https://twitter.com/hashtag/inclusion/">#inclusion</a></u>
   </b>. In recent years I have trended on GitHub with the likes of Twitter, Facebook, Angular etc and was in the top 500 open source contributors in the world. As a senior software engineer, the more I learn, the more I realise how much I still need to learn.</p>
```

**Comment by eddiejaoude** [comment]
Oh ok. As you prefer, same PR or new one I don't mind - thank you for the contributions 🤓

**Pull request closed without merging**